### ISSUE TITLE
Unable to use ONNX model exported by Optimum

### ISSUE THREAD

**Issue by lawrenceadams** [Anush008/fastembed-rs#81]
I wish to use model on huggingface that exist as PyTorch.bin files, however `fastembed-rs` does not support these if using [optimum to convert these](https://huggingface.co/docs/optimum/en/exporters/onnx/usage_guides/export_a_model).

Optimum tends to export the last layer as `logits` and not `last_hidden_state`: but at present fastembed-rs is hardcoded to use the latter.

Desired outcome is to be able to use optimum generated ONNX models to use this tool, allowing many more models: e.g. [BioBERT](https://huggingface.co/dmis-lab/biobert-base-cased-v1.2).
